vagrant/roles/candlepin-root/tasks/main.yml

```diff
@@ -19,24 +33,94 @@
     - python-psycopg2
     - libxml2-python
     - wget
+  tags:
+    - candlepin-root
+
+- name: Ensure that all packages are up2date
+  package:
+    name: "*"
+    state: latest
+  tags:
+    - candlepin-root
+    - system_update
+
 - name: Add remote debugging config to tomcat
-  lineinfile: dest=/etc/tomcat/tomcat.conf line="JAVA_OPTS=-Xdebug -Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n"
+  lineinfile:
+    dest: /etc/tomcat/tomcat.conf
+    line: "JAVA_OPTS=-Xdebug -Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n"
+  tags:
+    - candlepin-root
+
 - name: Initiate database
   command: service postgresql initdb
-           creates=/var/lib/pgsql/data/postgresql.conf
-- replace: dest=/var/lib/pgsql/data/pg_hba.conf regexp='(ident|peer)' replace='trust' backup=yes
+  args:
+    creates: /var/lib/pgsql/data/postgresql.conf
+  tags:
+    - candlepin-root
+
+- name: Set postgresql to trust
+  replace:
+    dest: /var/lib/pgsql/data/pg_hba.conf
+    regexp: '(ident|peer)'
+    replace: 'trust'
+    backup: yes
+  tags:
+    - candlepin-root
+
 - name: Start PostgreSQL and enable at boot
-  service: name=postgresql
-           enabled=yes
-           state=started
+  service:
+    name: postgresql
+    enabled: yes
+    state: started
+  tags:
+    - candlepin-root
+
+- name: Check if firewalld is on
```

Comment:
Added this check to see if firewalld is active on the system. If it is not, the "Punch holes in firewall" task below will skip.